index.js

```diff
@@ -92,3 +92,11 @@ var PUBLISH_SCRIPT =
   'for i = 2, #ARGV do ' +
     'redis.call("publish", ARGV[i], ARGV[1]) ' +
   'end';
+
+function close(client) {
+  if (client.close) {
```

Comment:
I think technically `.quit()` is [only deprecated for Redis server >=7.2](https://redis.io/docs/latest/commands/quit/):

> As of Redis version 7.2.0, this command is regarded as deprecated.
>
> It can be replaced by just closing the connection when migrating or writing new code.

I think I'd read this notice as anyone using Redis server <7.2 should still use `.quit()`, regardless of Node.js client version(?).

Not sure how we want to handle this?

Reply:
@alecgibson @ericyhwang 
This is a breaking change anyway:
- https://github.com/share/sharedb-redis-pubsub/pull/25

Maybe we can just drop support for node-redis < 5.0.0 then we can just use `close`.

Reply:
I think we shouldn't make this change breaking. It's nice to let people use the older version if possible, especially because it makes the migration path easier to keep the shim around. And since it's super easy for us, I don't see why not.